pkgs/stdenv/freebsd/bootstrap-files/x86_64-unknown-freebsd.nix

```diff
@@ -0,0 +1,13 @@
+{
+  unpack = import <nix/fetchurl.nix> {
+    url = "http://192.168.122.1:8000/result/on-server/unpack.nar.xz";
```

Comment:
Note for myself: This should be replaced with tarballs.nixos.org

Reply:
Well, now it's blocking `nixpkgs-unstable` channel, because you added a blocker that can't even fetch, let alone build.

Reply:
Example on Hydra: https://hydra.nixos.org/build/267353804